package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "rails-template",
```

Comment:
I recommend using `npx sort-package-json` to sort this - that's what I loosely recommend we use, but am not too focused on enforcement as it's a minor nicety vs `eslint` & co.